pytrustfall/pyproject.toml

```diff
@@ -1,11 +1,13 @@
+[build-system]
+requires = ["maturin>=1.6,<2"]
+build-backend = "maturin"
```

Comment:
Would you mind keeping this block at the bottom, in its original spot? It's mostly only useful to tools, and I'd like to keep the file order organized by how useful the content is to humans :)